Makefile

```diff
@@ -15,11 +15,10 @@ vendor: glide.yaml
 	# To build without Docker just run "glide install -strip-vendor"
 	docker run --rm \
     -v $(CURDIR):/go/src/github.com/$(PACKAGE_NAME):rw \
-    -v $(HOME)/.glide:/home/user/.glide:rw \
```

Comment:
Why removing the caching capabilities of this?  Doesn't this greatly slow down the build?